client/state/ui/action-log/reducer.js

```diff
@@ -16,7 +16,12 @@ import {
 	SITE_SETTINGS_RECEIVE,
 } from 'state/action-types';
 
+const relevantAnalyticsEvents = [
+	'calypso_themeshowcase_theme_click',
+];
+
 const relevantTypes = {
+	ANALYTICS_EVENT_RECORD: isRelevantAnalytics,
```

Comment:
I came back to this today and, in the spirit of YAGNI, tried something like:

```js
const relevantTypes = {
	EDITOR_PASTE_EVENT,
	…
};

const relevantAnalyticsEvents = [
	'calypso_themeshowcase_theme_click',
];

const isRelevantAnalytics = ( action ) =>
	get( action, 'meta.analytics', [] ).some( record =>
		includes( relevantAnalyticsEvents, record.payload.name ) );

const isRelevantAction = ( action ) =>
	has( relevantTypes, action.type ) ||
	isRelevantAnalytics( action );
```

That is, getting rid of any mention of `ANALYTICS_EVENT_RECORD` — which is fitting, since analytics are to be found in `meta` in general, and not in actions of specific types — and getting rid of the predicate mechanism.

However, this may create the wrong incentives for future users of `action-log`, leading either to catch-all action types that get matched too often, or to custom logic being added as yet another exception in the reducer file.

So I'll stick with the current approach of having mixed strings and predicates in an object, though I'll get rid of mentions of `ANALYTICS_EVENT_RECORD` and only look for `meta.analytics` instead.